src/components/bot-challenge-protection/index.jsx

```diff
@@ -29,7 +29,7 @@ const BotChallengeProtection = ({
 			/* eslint-disable-next-line */
 			recaptchaRef.current.reset();
 		}
-	}, [captchaError, error, isRecaptchaEnabled, recaptchaRef, recaptchaVersion, resetRecaptcha]);
```

Comment:
Removing this will cause a lint error, and rightly so. The `recaptchaRef` is defined outside of the `useEffect` so it needs to be part of the dependency array.